src/main/java/de/srendi/advancedperipherals/common/util/ChunkManager.java

```diff
@@ -65,12 +66,10 @@ public static void afterServerStarted(ServerStartedEvent event) {
     }
 
     @SubscribeEvent
-    public static void serverTick(TickEvent.ServerTickEvent event) {
-        if (event.phase == TickEvent.Phase.END) {
+    public static void serverTick(ServerTickEvent.Pre event) {
```

Comment:
so from listening tick END event, now we are listening tick START event, I want to confirm that's what you really want.

Reply:
Thanks for noticing, will keep it at post